lib/signal-list-view.js

```diff
@@ -53,7 +53,7 @@ export default class SignalListView {
     const kernel = store.kernel;
     if (!kernel) return;
     const commands =
-      kernel instanceof WSKernel
+      kernel.kernel instanceof WSKernel // XXX(nikita): breaks abstraction
```

Comment:
Ditto: I'm not sure what's the intention of this comment? This is guarded by `kernel.kernel instanceof WSKernel`.